docs/data_usage_guides/affected_people.md

```diff
@@ -2,29 +2,114 @@
 
 ---
 
+## Internally-Displaced Persons <a id=”idps”></a>
+
+The data for this sub-category is taken from the
+[International Organization for Migration](https://www.iom.int/) (IOM)'s
+[displacement tracking matrix](https://dtm.iom.int/) (DTM),
+which collects and analyzes data on population movements,
+displacements, and mobility patterns to provide timely information
+for humanitarian response efforts.
+THe DTM data used in HAPI is taken from their
```

Comment:
Capitalisation of "THe"

Reply:
I would have never spotted this in a million years

Reply:
I wouldn't have spotted it in my own writing!